runtime/src/main/java/org/corfudb/util/Utils.java

```diff
@@ -423,7 +424,7 @@ public static void sleepUninterruptibly(long millis) {
                 Thread.sleep(millis);
                 return;
             } catch (InterruptedException ie) {
-                log.trace("Interrupted");
+                throw new UnrecoverableCorfuInterruptedError("Uninterruptible sleep interrupted", ie);
```

Comment:
@no2chem Sorry I overlooked this in my review earlier.
But this method is not required now.
The java doc, the method name and the error are all contradicting each other.

Reply:
I'll update the doc description, but I think we should use this utility when we need to sleep, and being interrupted means that Corfu cannot recover.